lib/shelley/src/Cardano/Wallet/Shelley/Compatibility.hs

```diff
@@ -1347,31 +1351,28 @@ rewardAccountFromAddress (W.Address bytes) = refToAccount . ref =<< parseAddr by
 --
 -- See 'W.TokenBundleSizeAssessor' for the expected properties of this function.
 --
-tokenBundleSizeAssessor :: W.TokenBundleSizeAssessor
-tokenBundleSizeAssessor = W.TokenBundleSizeAssessor {..}
+tokenBundleSizeAssessor :: W.TokenBundleMaxSize -> W.TokenBundleSizeAssessor
+tokenBundleSizeAssessor maxSize = W.TokenBundleSizeAssessor {..}
   where
     assessTokenBundleSize tb
-        | serializedLengthBytes <= maxTokenBundleSerializedLengthBytes =
+        | serializedLengthBytes <= maxSize' =
             W.TokenBundleSizeWithinLimit
         | otherwise =
             W.OutputTokenBundleSizeExceedsLimit
       where
         serializedLengthBytes :: Int
         serializedLengthBytes = computeTokenBundleSerializedLengthBytes tb
 
+        maxSize' :: Int
+        maxSize' = fromIntegral
+            $ getQuantity
+            $ W.unTokenBundleMaxSize maxSize
+
 computeTokenBundleSerializedLengthBytes :: TokenBundle.TokenBundle -> Int
 computeTokenBundleSerializedLengthBytes =
+    -- FIXME: Do we need an Alonzo-specific verison as well?
```

Comment:
Shouldn't forget to check 👆but I suspect the answer is no.

Reply:
I checked `alonzo.cddl` and it's the same as `shelley-ma.cddl` for multiassets. And there is no `toAlonzoValue` in cardano-api, only `toMaryValue`.

Reply:
👍 Thanks!